### PR TITLE
Add GetStreamKey

### PR DESCRIPTION
**Pull request by kivutar**

### CODE REVIEW

File: streams.go [hunk 1]
```diff
@@ -82,3 +87,20 @@ func (c *Client) GetFollowedStream(params *FollowedStreamsParams) (*StreamsRespo
...
+// Required scope: channel:read:stream_key
+func (c *Client) GetStreamKey(params *StreamKeyParams) (*StreamsResponse, error) {
+	resp, err := c.get("/streams/key", &ManyStreams{}, params)
+	if err != nil {
+		return nil, err
+	}
+
+	streams := &StreamsResponse{}
+	resp.HydrateResponseCommon(&streams.ResponseCommon)
+	streams.Data.Streams = resp.Data.(*ManyStreams).Streams
```
Comment (nicklaw5) [inline streams.go:102]: `ManyStreams` is not a correct representation of the response object for this endpoint. The response, according to the [docs](https://dev.twitch.tv/docs/api/reference#get-stream-key), should only in the `stream_key` property.

```json
{
  "data": [
    {
      "stream_key": "live_44322889_a34ub37c8ajv98a0"
    },
  ]
}
```

My suggestion would be to create a new struct called `ManyStreamKeys` instead. For example:

```go
type ManyStreamKeys struct {
    Data []struct{
        StreamKey string `json:"stream_key"`
    } `json:"data"`
}
```

File: streams.go [hunk 2]
```diff
@@ -82,3 +87,20 @@ func (c *Client) GetFollowedStream(params *FollowedStreamsParams) (*StreamsRespo
...
+	streams.Data.Pagination = resp.Data.(*ManyStreams).Pagination
```
Comment (nicklaw5) [inline streams.go:103]: There's also no pagination for this endpoint according to the [docs](https://dev.twitch.tv/docs/api/reference#get-stream-key).

**Review by nicklaw5 — CHANGES_REQUESTED** [review]
Thanks for the PR @kivutar. Can you please review the comments I've left?

Also, we'll need some tests for this endpoint before I can merge.

**Comment by kivutar** [comment]
@nicklaw5 this should be fine now

**Review by nicklaw5 — APPROVED** [review]
Good enough for me :+1:

(Sorry for the delay)

**Pull request merged**